### PR TITLE
Objects Automated Test - LPS-152054 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**
https://issues.liferay.com/browse/LPS-152054

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro [hunk 1]
```diff
@@ -100,6 +100,14 @@ definition {
 			locator1 = "ObjectCustomViews#VIEW_BUILDER_EMPTY_STATE_TEXT");
 	}
 
+	macro assertNoColumnAddYet {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro:103]: Can you change this to assertNoColumnPresentInDefaultSort to make it clearer what we're asserting?

**Review by timpak — CHANGES_REQUESTED** [review]
@evanilson-santana looks good, just have a couple minor things to address. Can you rename the macro and squash your commits together? In the future, you can just add the SF changes to the original commit that caused the error.

**Comment by timpak** [comment]
See https://github.com/timpak/liferay-portal/pull/214#issuecomment-1129668804

**Pull request closed without merging**